servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -440,6 +423,25 @@ public Completable closeAsyncGracefully() {
                     .map(AsyncCloseable::closeAsyncGracefully)::iterator);
         }
 
+        @Override
+        public boolean equals(final Object o) {
```

Comment:
Doesn't look like we started to use equals and hashCode. Can we defer adding them for now?